frontend/src/components/ProofingTab/GraphDiagram/Widgets/DestinationNodeWidget.tsx

```diff
@@ -22,14 +24,20 @@ export const DestinationNodeWidget: React.FC<DestinationNodeProps> = ({
     const [zIndex, setZIndex] = useState(0);
 
     // Function to toggle the state
-    const toggleColor = () => {
-        setIsActive(!isActive);
-        setZIndex((prevZIndex) => prevZIndex + 1);
+    const toggleColor = (event: any) => {
+        if (event.shiftKey) {
+            setIsActive(!isActive);
+            setZIndex((prevZIndex) => prevZIndex + 1);
+        }
     };
 
 
     const inPort = model.getPort("in");
     const hasForwardConnections = model.getOptions()?.forward_connection?.length > 0;
+    
+    const isAfferentT = model.getOptions().anatomicalType === DestinationTypeMapping["AFFERENT-T"]
```

Comment:
TypeC11Enum.AfferentT here as well, we are already importing it in the component.